HackYourFuture-CPH/fp-class20
# Migrations

# Description
This PR adds the data migrations to the server side. 

#Story:
As a developer I want to be able to to see 6 tables according to the db diagram so that I can seed data. I build these scripts via the command line using knex command line tool.

Fixes # (issue)
https://hackyourfuture-dk.atlassian.net/jira/software/projects/CLASS20/boards/1/backlog?selectedIssue=CLASS20-25

# How to test?
1- 
**yarn knex migrate:up** should create the tables
**yarn knex migrate:down** should drop the tables
2- Make sure all the table are according to this DB diagram: https://dbdiagram.io/d/616ae287940c4c4eec9881e3




# Checklist
